### ISSUE TITLE
[Issue] 'Unload' button not shown when AMS is plugged in (This is a firmware issue, but there is no channel to report firmware issues on this server)

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/BambuStudio#913]
After using the back spool holder on X1, the 'Unload' button doesn't show on the touch screen to unload the filament, if the AMS is still plugged in. The button will show only after AMS is unplugged.

**Comment by Merlwynd** [comment]
This is the same as issue #766.....there has been no response to issue 766 yet......

**Comment by Benderrodrigu3z** [comment]
heat the nozzle and hit extrude button the screen, its the Up arrow on the movement screen 
![Screen Shot 2022-11-20 at 12 38 31 AM](https://user-images.githubusercontent.com/48887257/208586476-6c556002-d360-462b-9049-448a8b8ca509.png)

**Comment by Merlwynd** [comment]
of course....or Bambu Lab could have the 'Unload" button reappear whenever the "Enable AMS" button is unchecked (meaning you are using an external spool) when sending a print.....so you don't have to do all those steps to unload the external filament....
